ui/src/pages/Login/SignIn/SignIn.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from "react";
+
+export function SignIn() {
+  const [username, setUsername] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
+  const [email, setEmail] = useState<string>("");
+
+  const handleUsernameChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setUsername(event.target.value);
+  };
+
+  const handlePasswordChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(event.target.value);
+  };
+
+  const fetchRegister = async () => {
+    try {
+      const response = await fetch("/auth/register", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json", // Set Content-Type header
+        },
+        body: JSON.stringify({
+          username: username,  // Use state values for the username and password
+          email: email,
+          password: password
+        }),
+      });
+      if (!response.ok) {
+        throw new Error(`HTTP error! status: ${response.status}`);
+      }
+      const result = await response.json();
+      console.log("Username: ", username, "\nPassword: ", password, "\nResponse: ", result);
+    } catch (error) {
+      console.error("Error fetching data:", error);
+    }
+  };
+  return (
+    <>
+      <>
+        <img src="/images/terraria-bunny1.gif" className="login__bunnies" />
+        <input
+          className="login__input"
+          placeholder="Username or Email"
+          value={username}
+          onChange={handleUsernameChange}
+        />
+        <img
+          src="/images/terreria-bunnyFLIPPEd.gif"
+          className="login__bunnies"
+        />
+        <input
+          className="login__input"
+          placeholder="Password"
+          value={password}
+          onChange={handlePasswordChange}
+        />
```

Comment:
for this input can you make it so the password blurs as it gets typed

Reply:
este wey como sigue con su pinche "blur password" mamadas

Reply:
fine

Reply:
Lolololol it just feels half assed if pushed up and this feature isn't there considering it is an essential part to the component. Like if it were a "show password" check box I would understand to put it into a whole new ticket since it isn't necessary for the component function

Reply:
Sure, but thats not the point of this ticket. We are doing this incrementally. Having a component like this wouldn't be creating it, it would be adding new features to it. Thats something we want to leave for a differetn ticket. Besides, we would need to talk about what "essential" means in this context. Hiding the password isnt "essential" to having a page be created. Its a feature and NOT mandatory for a page to just exist

Reply:
> Sure, but thats not the point of this ticket. We are doing this incrementally. Having a component like this wouldn't be creating it, it would be adding new features to it. Thats something we want to leave for a differetn ticket. Besides, we would need to talk about what "essential" means in this context. Hiding the password isnt "essential" to having a page be created. Its a feature and NOT mandatory for a page to just exist

mhmm in that case this ticket would have been done with a bare bones button and text field no styling needed yet there is. as it isn't 'essential' in your styling it is a key characteristic of a password text field. do it and merge this ticket in.

Reply:
There is a difference between having styling be part of the pr and having a feature such as that included with functionality. Things like that require actual logic im assuming to implement, if its easy to do then ill do it in this ticket. But if itll take logic and functions, then thats a different ticket